subprojects/core/src/main/java/org/gradle/api/internal/tasks/properties/ModifierAnnotationCategory.java

```diff
@@ -46,6 +47,9 @@
     ),
     IGNORE_EMPTY_DIRECTORIES("ignore empty directories",
         IgnoreEmptyDirectories.class
+    ),
+    IGNORE_LINE_ENDINGS("ignore line endings",
+        IgnoreLineEndings.class
```

Comment:
This should probably be `@NormalizeLineEndings`, too, no?

Reply:
Changed.